_data/software/iperf.yml

```diff
@@ -0,0 +1,36 @@
+---
+schemaVersion: '1.0'
+description:
+  whatItDoes:
+    en: >-
+      iPerf is a tool for active measurements of the maximum achievable
+      bandwidth on IP networks. 
+    fr: >-
+      iPerf est un outil de mesure active de la bande passante maximale
+      réalisable sur les réseaux IP.
+category: '43232900'
+homepageURL:
+  en: 'https://iperf.fr/'
```

Comment:
The English site is on .fr and the French wiki is on .info https://lafibre.info/iperf/ :)

Reply:
Not sure we want to redirect to a Forum/Wiki for the French text as it may just end up confusing folks. Hard to say. Also super weird for a .fr domain to be english only.

Reply:
Nope, I wouldn't point to the forum, just found the domain choices odd